client/blocks/happychat/chat-panel/style.scss

```diff
@@ -0,0 +1,88 @@
+/**
+ * Base styles
+ */
+
+.chat-panel {
+	overflow: hidden;
+	position: fixed;
+	bottom: 0;
+	right: 12px;
+	font-size: 14px;
+	display: none;
+	box-sizing: border-box;
+	height: 330px;
+
+	transform: translateZ(0);
+	backface-visibility: hidden;
+
+	&.is-open {
+		display: block;
+		background: $gray-lighten-30;
+		bottom: 0;
+		z-index: z-index( 'root', '.chat-panel.is-open' );
+	}
+
+	&.is-hidden {
+		display: none;
+	}
+}
+
+/**
+ * Panel mode
+ */
+@include breakpoint( ">480px" ) {
+
+	.layout:not( .is-section-chat ) {
+		.chat-panel.is-open {
+			box-shadow: 0 1px 2px rgba( 0,0,0,.2 ), 0 1px 10px rgba( 0,0,0, .1 );
+			width: 280px;
+		}
+	}
+
+}
+
+@include breakpoint( "<480px" ) {
```

Comment:
According to our [css styling guidelines](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/css.md#the-dry-principle), we should use the `breakpoint` mixin from `assets/stylesheets/shared/mixins/_breakpoints.scss` instead of writing our own breakpoints.

Reply:
Good catch!